Lib/tkinter/__init__.py

```diff
@@ -3965,7 +3965,7 @@ def __init__(self, master, variable, value, *values, **kwargs):
         if 'command' in kwargs:
             del kwargs['command']
         if kwargs:
-            raise TclError('unknown option -'+kwargs.keys()[0])
+            raise TclError('unknown option -'+next(iter(kwargs.keys())))
```

Comment:
`.keys()` is not needed.
```suggestion
            raise TclError('unknown option -'+next(iter(kwargs)))
```

Reply:
I preferred being explicit here, but I'm happy with it either way.